scriptmodules/emulators/amiberry.sh

```diff
@@ -106,12 +106,12 @@ function configure_amiberry() {
 
     # move config / save folders to $md_conf_root/amiga/amiberry
     local dir
-    for dir in conf savestates screenshots; do
+    for dir in conf nvram savestates screenshots; do
         moveConfigDir "$md_inst/$dir" "$md_conf_root/amiga/amiberry/$dir"
     done
 
     # symlink cd32.nvr from $md_inst/data to "$md_conf_root/amiga/amiberry
-    moveConfigFile "$md_inst/data/cd32.nvr" "$md_conf_root/amiga/amiberry/cd32.nvr"
+    moveConfigFile "$md_inst/data/cd32.nvr" "$md_conf_root/amiga/amiberry/nvram/cd32.nvr"
```

Comment:
From this change it looks like any existing file at "$md_conf_root/amiga/amiberry/cd32.nvr" will not be used after this change, so you may want to move that file to the new location before updating the symlink.

Reply:
Thanks! I've made a new change hoping I have it right this time. The new `nvram` location is now fully functional in Amiberry and this is where any CDTV/CD32 nvram files must be stored.